config/locales/en/mailers.yml

```diff
@@ -88,6 +88,9 @@ en:
       allowance_will_renew: >-
         %{prisoner} will have their full visiting allowance (VO) renewed on
         %{date}.
+      no_adult:
```

Comment:
This will need welsh translation, let me know what I need to do. I haven't got Transifex configured on my machine